apps/web/utils/tailwindHelper.ts

```diff
@@ -4,20 +4,20 @@ import { converter, formatRgb } from 'culori/fn';
 const rgb = converter('rgb');
 
 const spectrumToTailwindMapper = [
-  { id: '-5', weight: '50' },
-  { id: '-4', weight: '100' },
-  { id: '-3', weight: '200' },
-  { id: '-2', weight: '300' },
-  { id: '-1', weight: '400' },
+  { id: '+5', weight: '50' },
+  { id: '+4', weight: '100' },
+  { id: '+3', weight: '200' },
+  { id: '+2', weight: '300' },
+  { id: '+1', weight: '400' },
   { id: '0', weight: '500' },
-  { id: '+1', weight: '600' },
-  { id: '+2', weight: '700' },
-  { id: '+3', weight: '800' },
-  { id: '+4', weight: '900' },
-  { id: '+5', weight: '950' },
+  { id: '-1', weight: '600' },
+  { id: '-2', weight: '700' },
+  { id: '-3', weight: '800' },
+  { id: '-4', weight: '900' },
+  { id: '-5', weight: '950' },
 ];
 
-const spectrumToTailwind = (spectrumList: SpectrumList) => {
+export const spectrumToTailwind = (spectrumList: SpectrumList) => {
```

Comment:
```suggestion
const spectrumToTailwind = (spectrumList: SpectrumList) => {
```
**todo**: Not used elsewhere.